app/models/katello/repository.rb

```diff
@@ -486,6 +492,25 @@ def import_system_applicability
       end
     end
 
+    def self.import_distributions
+      self.all.each do |repo|
+        repo.import_distribution_data
+      end
+    end
+
+    def import_distribution_data
+      distribution = Katello.pulp_server.extensions.repository.distributions(self.pulp_id).first
+      if distribution
+        self.update_attributes!(:distribution_version => distribution["version"],
+                                :distribution_arch => distribution["arch"],
+                                :distribution_family => distribution["family"],
+                                :distribution_variant => distribution["variant"],
+                                :distribution_uuid => distribution["_id"],
+                                :distribution_bootable => ::Katello::Repository.distribution_bootable?(distribution)
+                                )
```

Comment:
If leaving the closing parent by itself, I think it better to just use a hash style indentation strategy:

```
self.update_attributes!(
  :distribution_version => distribution["version"],
  :distribution_arch => distribution["arch"],
  :distribution_family => distribution["family"],
  :distribution_variant => distribution["variant"],
  :distribution_uuid => distribution["_id"],
  :distribution_bootable => ::Katello::Repository.distribution_bootable?(distribution)
)
```